src/components/shared/map/Map.tsx

```diff
@@ -160,30 +160,49 @@ const Map = ({
     }
   }, [mapRef.current]);
 
+  const saveMapState = () => {
+    if (mapRef.current) {
+      console.log('called handle navigation');
+      const currentCenter = mapRef.current.getCenter();
```

Comment:
Hey @adisa39 - It appears I am getting the following when I go back to the map after saving my 'Search Center' ->
![image](https://github.com/user-attachments/assets/bfc94415-7764-4b7b-959f-f32f1c0d9235)